### PR TITLE
Updates Syntax for DEVELOPMENT-SNAPSHOT-2016-07-25-a

### PR DESCRIPTION
**Pull request by barrault01**
This PR is updating Postgresql to the same swift version of the main vapor repo : DEVELOPMENT-SNAPSHOT-2016-07-25-a. 
I also add a .gitignore to ignore build files and xcodeproj.

Also fixed unit tests.

### CODE REVIEW

File: Tests/PostgreSQL/PostgreSQLTests.swift [hunk 1]
```diff
@@ -87,7 +87,7 @@ class PostgreSQLTests: XCTestCase {
                 XCTFail("Could not get pi result")
             }
 
-            if let result = try postgreSQL.execute("SELECT * FROM parameterization WHERE u = $1", [.int(42)]).first {
+            if let result = try postgreSQL.execute("SELECT * FROM parameterization WHERE u = $1", [.string("42")]).first {
```
Comment (tanner0101) [inline Tests/PostgreSQL/PostgreSQLTests.swift:90]: Why was this changed to string?

File: Sources/PostgreSQL/Database.swift [hunk 2]
```diff
@@ -39,17 +39,17 @@ public class Database {
         }
         
         guard !query.isEmpty else {
-            throw Error.noQuery
+            throw DatabaseError.noQuery
         }
         
         let res: Result.ResultPointer
-        if let values = values where values.count > 0 {
+        if let values = values , values.count > 0 {
```
Comment (tanner0101) [inline Sources/PostgreSQL/Database.swift]: The commas here should not have a space before them.

**Comment by tanner0101** [comment]
Other than lack of CI on this repo, the PR looks good. :) Can you confirm tests are passing on your machine after the changes noted?

File: Tests/PostgreSQL/PostgreSQLTests.swift (hunk 1 above)
Comment (barrault01) [inline Tests/PostgreSQL/PostgreSQLTests.swift:90]: If I keep as int the tests were failling.

**Comment by barrault01** [comment]
The tests passing: 
Tests passing

**Comment by barrault01** [comment]
So as travis integration are still failling but is passing on circle ci. I think it will be ok to remove travis integration for the moment.
